physics/module_bl_mynn.F90

```diff
@@ -3863,28 +3864,28 @@ SUBROUTINE  mym_condensation (kts,kte,   &
            !Add condition for falling/settling into low-RH layers, so at least
            !some cloud fraction is applied for all qc, qs, and qi.
            rh_hack= rh(k)
-           !ensure adequate RH & q1 when qi is at least 1e-9
-           if (qi(k)>1.e-9) then
-              rh_hack =min(1.0_kind_phys, rhcrit + 0.07*(9.0 + log10(qi(k))))
+           !ensure adequate RH & q1 when qi is at least 1e-9 (above the PBLH)
+           if (qi(k)>1.e-9 .and. zagl .gt. pblh2) then
```

Comment:
This is the change currently causing the same failure as in https://github.com/ufs-community/ccpp-physics/issues/114.

Reply:
I don't think there is anything wrong with that line of code. This just seems to be (one of many) tweaks that can apparently push the model over the edge. You aren't suggesting this is actually the culprit, right?

Reply:
Right, I'm just saying that this line is the "straw that breaks the camel's
back" for this PR with respect to the known issue, and removing this change
would cause all RTs to at least run to completion, which, as you know, is
an arbitrary bar. So, in that sense, this is the proximate cause and not
the ultimate cause.